rpcs/ledgerService.go

```diff
@@ -259,6 +287,23 @@ func (ls *LedgerService) sendCatchupRes(ctx context.Context, target network.Unic
 	}
 }
 
+func topicBlockBytes(ledger *data.Ledger, round basics.Round, requestType string) network.Topics {
+	blk, cert, err := ledger.EncodedBlockCert(round)
+	if err != nil {
+		return network.Topics{
+			network.MakeTopic(network.ErrorKey, []byte(err.Error()))}
+	}
+	if requestType == network.BlockAndCertValue {
```

Comment:
nit: use switch-case here, and if the requestType is not supported, return an error code.